src/input/composite_device/mod.rs

```diff
@@ -785,6 +807,12 @@ impl CompositeDevice {
             return Ok(());
         }
 
+        // If force feedback is disabled at the composite device level, don't
+        // forward any other FF events to source devices.
+        if !self.ff_enabled && event.is_force_feedback() {
+            return Ok(());
+        }
+
```

Comment:
Side note: this should be doable now.

Reply:
Uhg, not showing the line I selected to comment. Refers to Todo for routing events only to capable devices